moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -152,13 +152,12 @@ class ServoCalcs
    * Is handled differently for position vs. velocity control.
    */
   void suddenHalt(trajectory_msgs::msg::JointTrajectory& joint_trajectory) const;
-  void suddenHalt(sensor_msgs::msg::JointState& joint_state) const;
 
   /** \brief  Scale the delta theta to match joint velocity/acceleration limits */
   void enforceVelLimits(Eigen::ArrayXd& delta_theta);
 
   /** \brief Avoid overshooting joint limits */
-  bool enforcePositionLimits(sensor_msgs::msg::JointState& joint_state) const;
+  bool enforcePositionLimits(trajectory_msgs::msg::JointTrajectory& joint_trajectory) const;
```

Comment:
You may be overwriting this commit:
https://github.com/ros-planning/moveit2/commit/fb94e78da7d9a16fee53ccc34fbfb2c7af2af5d0